cmd/utils/flags.go

```diff
@@ -189,7 +189,7 @@ var (
 		Name:  "lightkdf",
 		Usage: "Reduce key-derivation RAM & CPU usage at some expense of KDF strength",
 	}
-	// Dashboard settings
```

Comment:
These changes don't improve things because they replace each "section heading" style comment with a comment about the first flag in the section. IMHO it would be better to add a newline after the section instead, e.g.

```go
 	// Dashboard settings

 	DashboardEnabledFlag = cli.BoolFlag{
 		Name:  "dashboard",
 		Usage: "Enable the dashboard",
 	}
 	DashboardAddrFlag = cli.StringFlag{
 		Name:  "dashboard.addr",
 		Usage: "Dashboard listening interface",
 		Value: dashboard.DefaultConfig.Host,
	}
```